Dockerfile

```diff
@@ -1,31 +1,47 @@
+# Stage 1 build
 FROM golang:1.16.1-buster AS builder
+RUN apt-get update && apt-get install ca-certificates -y
+ENV CGO_ENABLED=0
+
+ENV USER=bubbly
+ENV UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/nonexistent" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    "${USER}"
 
-WORKDIR /src
+WORKDIR $GOPATH/src/main/bubbly/
```

Comment:
Nit: the notation makes me think the distro uses `GOPATH` in its original way [1]. If so, should this not be a fully-qualified name?

As in `$GOPATH/src/github.com/valocode/bubbly`?

This is a nit and I'm just wondering, not necessarily requesting change.

[1]: [How to Write Go Code (with GOPATH) | golang.org](https://golang.org/doc/gopath_code)

Reply:
Good point! Always good to be more precise